stric-react/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@bun-examples/stric-react",
+  "module": "index.ts",
+  "devDependencies": {
+    "bun-types": "^0.5.0",
```

Comment:
maybe i would use `latest` here as well

bun is at a stage where everyone is upgrading right away, not like node where you have people using v16, v18, v19